docs/elasticsearch-spec.asciidoc

```diff
@@ -22,6 +22,24 @@ Advanced settings
 - Pod disruption budget
 - Change budget (maxUnavailable, maxSurge)
 
+[id="{p}-jvm-heap-size"]
+=== JVM heap size
+
+By default, JVM heap used by Elasticsearch has minimum and maximum size set to 1 GB. You can change that by using `ES_JAVA_OPTS` environment variable as shown below. For more information on setting the heap size, see link:https://www.elastic.co/guide/en/elasticsearch/reference/current/heap-size.html[Setting the heap size].
```

Comment:
I think we could mention the default mem request.

"... set to 1 GB with a default [memory request](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#meaning-of-memory) of 2Gi (no memory limit):"

And maybe explain the good pratice to keep a factor 2 between the JVM heap size and the memory resources requested.